media-libs/assimp/files/assimp-5.2.3-drop-Werror-gcc-option.patch

```diff
@@ -0,0 +1,22 @@
+From dea8b4d8c533c131bd546c316f8df8af8897ac75 Mon Sep 17 00:00:00 2001
```

Comment:
I'll handle it here but remember to include bug links etc (both gentoo ones and any upstream stuff, or patch links) at top of patches to make it easier for future research

Reply:
To what should I change this, if there's no link to upstream or gentoo?

Reply:
Well, there's at least one Gentoo link (the bug you filed), but there's also the bug you filed upstream?

Reply:
Ok, I see. I always understood this in way, that I should change the from line and point to a valid git commit, which makes sense in cases where I pull a patch from upstream or somewhere else, but not in the case, where I create a patch myself.
I will add such links in future patches. Thanks for the clarification.